src/gui/src/Mixer/MixerLine.cpp

```diff
@@ -822,11 +826,11 @@ void MasterMixerLine::rotaryChanged( Rotary *pRef )
 
 	if ( pRef == m_pHumanizeTimeRotary ) {
 		pEngine->getSong()->setHumanizeTimeValue( fVal );
-		sMsg = tr( "Set humanize time parameter [%1]").arg( fVal, 0, 'f', 2 );
+		sMsg = tr( "Set hum. time parameter [%1]").arg( fVal, 0, 'f', 2 ); //not too long for display
```

Comment:
I stared at this for an embarrassingly long period of time before realising I could just look to the left to see what 'hum.' is an abbreviation for.

Rather than shortening 'humanize', could we instead shorten 'parameter' to 'param'? This is a well established abbreviation that I think would make the message more readable?